packages/shared-ux/page/solution_nav/src/solution_nav.tsx

```diff
@@ -100,7 +101,7 @@ export const SolutionNav: FC<SolutionNavProps> = ({
 }) => {
   const isSmallerBreakpoint = useIsWithinBreakpoints(mobileBreakpoints);
   const isMediumBreakpoint = useIsWithinBreakpoints(['m']);
-  const isLargerBreakpoint = useIsWithinBreakpoints(['l', 'xl']);
+  const isLargerBreakpoint = useIsWithinMinBreakpoint('l');
```

Comment:
what does this do? and where did `xl` go?

Reply:
With the refactored breakpoint hooks, we discovered a situation where the side nav would stop rendering if the browser window was 1600px or wider. By switching to the `useIsWithinMinBreakpoint('l')` we removed that upper range and effectively said at large breakpoint or wider, always evaluate to true.

Reply:
commit message reference: https://github.com/elastic/kibana/pull/140323/commits/cf858f152fc8a327aa1c46f1d31f4c5b15f562f6

> because of [apm/ux/uptime/synthetic's] new custom xxl/xxxl breakpoints, the `isLargerBreakpoint` logic was broken on the shared solution nav. switching to a min breakpoint bound fixes the issue and futureproofs the solution nav for all apps that add larger breakpoints than xl

For more context: EUI now supports both customizing our existing breakpoint sizes (xs through xl) _and_ adding more custom breakpoints (in this case, `xxl: 1600` and `xxxl: 2000` for the apm/ux/uptime/synthetic plugins.

This means that using `useIsWithinBreakpoints['l', 'xl']` is unintentionally too limiting for plugins with custom larger breakpoints, it translates to `is within the l and xl screens only` whereas this logic **really** wants `is l and above`, which is what `useIsWithinMinBreakpoint` does.